### PR TITLE
Create command for comparing across multiple nodes

### PR DESCRIPTION
**Pull request by aludvik**
Do `sawnet compare --help` for usage.

### CODE REVIEW

**Comment by aludvik** [comment]
There are a few gross things in here that I need to fix, but Andi should be able to use it as is

**Comment by chenette** [comment]
I believe that the new "sawnet" CLI should be documented in the CLI Reference.  You can follow the model of other parent-commands-with-subcommands, such as "sawtooth block" or "sawset". I'm available to help (aka edit the doc information).

**Comment by aludvik** [comment]
Fixed gross things

**Comment by agunde406** [comment]
Do test you actually have to do `sawnet compare-chains --help`

File: cli/sawtooth_cli/network_command/compare.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+        for block_id, node_ids in siblings.items():
+            for node_id in node_ids:
+                node_list[node_id] = block_id[:8]
+
+    # Print the last one
+    print(format_str.format(prev_block_num, *node_list))
+
+
+def print_tree(graph, tails):
```
Comment (agunde406) [inline cli/sawtooth_cli/network_command/compare.py]: Remove reference to --tree if it is not implemented

Comment (peterschwarz) [reply cli/sawtooth_cli/network_command/compare.py]: Also, delete the function

**Comment by aludvik** [comment]
Added an okay implementation of `--tree`. I have a wip commit that is approaching something better, but don't want to spend anymore time on this for now.

**Pull request merged**